File: JavaPrograms/FactorialFor.java
public class FactorialFor {
	public static void main(String args[]){
		factorial(6);
	}

	public static void factorial(int n) {
		int result = 1;
		for(int i=2; i<=n; i++) {
			result = result * i;
		}
		System.out.println(result);
	}
}

File: PythonPrograms/FactorialFor.py
def factorialfor(number):
    factorial = 1
    for x in range(1, number + 1):
        factorial = factorial * x
    print('The factorial of', number, ' is', factorial)

factorialfor(5)


# number = 5
# factorial   1   2   6   24  120
# x           1   2   3   4   5